examples/cpp/dds/TypeIntrospectionExample/arg_configuration.h

```diff
@@ -0,0 +1,252 @@
+// Copyright 2022 Proyectos y Sistemas de Mantenimiento SL (eProsima).
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @file arg_configuration.h
+ *
+ */
+
+#ifndef _EPROSIMA_FASTDDS_EXAMPLES_CPP_DDS_TYPEINTROSPECTIONEXAMPLE_ARG_CONFIGURATION_H_
+#define _EPROSIMA_FASTDDS_EXAMPLES_CPP_DDS_TYPEINTROSPECTIONEXAMPLE_ARG_CONFIGURATION_H_
+
+#include <iostream>
+#include <string>
+
+#include <optionparser.hpp>
+
+namespace option = eprosima::option;
+
+constexpr const char* HELLO_WORLD_DATA_TYPE_ARG = "hw";
+constexpr const char* ARRAY_DATA_TYPE_ARG = "array";
+constexpr const char* SEQUENCE_DATA_TYPE_ARG = "seq";
+constexpr const char* STRUCT_DATA_TYPE_ARG = "struct";
+constexpr const char* PLAIN_DATA_TYPE_ARG = "plain";
+constexpr const char* SIMPLELARGE_DATA_TYPE_ARG = "simple";
```

Comment:
Let`s use consistent names and call them equal to their folders 
```suggestion
constexpr const char* SIMPLELARGE_DATA_TYPE_ARG = "simple_large";
```